<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categorizedialog</class>
 <widget class="QDialog" name="categorizedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/categorize.png</normaloff>:/images/categorize.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
	background-image: url(:/images/loginbackground.jpeg);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="styleSheet">
       <string notr="true">font: 700 24pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>分类域名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QPushButton" name="addButton">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>40</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>添加分类类别</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="changeButton">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>40</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>修改域名分类</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QtMaterialAutoComplete" name="domainLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>输入域名. . .</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QtMaterialAutoComplete" name="typeLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>输入类别. . .</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QtMaterialAutoComplete" name="contentLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>输入类别内容. . .</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QtMaterialAutoComplete" name="deleteLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>输入. . .</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="deleteButton">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>40</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>删除分类类别</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QtMaterialAutoComplete" name="addLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>输入. . .</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
